docs/en/enterprise-edition/content-collections/administration/create-and-manage-saved-views.adoc

```diff
@@ -0,0 +1,43 @@
+== Create and Manage Saved Views
+
+Prisma Cloud offers the ability to create *Saved Views* to prioritize and organize your data into categories that are meanigful to your organization. Saved views are available on the Governance, Alerts, Compliance and Dashboards pages. For instance on the Alerts page Prisma Cloud provides a set of default views — Overview, Misconfigurations, Incidents, Malware, and Highest Priority, but you can also create a custom view and share it with others in your organization or tenant. Review the Alerts page workflows below to create, manage, and share your Saved Views.
```

Comment:
typo - please update "meaningful".
Add a comma after Compliance, and Dashboard pages.
Add a comma after "For instance", on the Alerts page,